.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+### Context
+
+Why are you making this change? What might surprise someone about it?
+
+### Changes proposed in this pull request
+
+If there are UI changes, please include Before and After screenshots.
```

Comment:
I do not think is this context, that including screenshots (especially Before) is important.
These are just docs, not complex UI.

Reply:
I can understand your point. Do you think it should be removed or better worded?

Reply:
We discussed IRL that this should be removed.